dev/k8s/dpf-sim-controller/Makefile

```
@@ -0,0 +1,95 @@
+.PHONY: image
+image: docker-build docker-push ## Build + push the image at IMG
+
+.PHONY: install-crds
+install-crds: ## Install ONLY the DPF CRDs the simulator drives (never the operator)
+	kubectl apply -f $(CRD_DIR)
+	# apply returns before discovery serves the new types; starting the
+	# manager that early fails its cache/watch init on a fresh cluster.
+	kubectl wait --for condition=established --timeout=60s -f $(CRD_DIR)
+
+.PHONY: deploy
+deploy: install-crds ## One-command deploy: CRDs + RBAC + Deployment (override IMG=, DPF_NAMESPACE=, PULL_SECRET=)
```

checkmake (0.3.2)

[warning] Target body for "deploy" exceeds allowed length of 5 lines (13).

(maxbodylength)